packages/code-du-travail-frontend/static/assets/icons/external-link.svg

```diff
@@ -1 +1,10 @@
-<svg xmlns="http://www.w3.org/2000/svg" width="13.93cm" height="13.4cm" viewBox="0 0 394.91 379.91"><defs><style>.cls-1{fill:#4c5467;}</style></defs><title>Plan de travail 1</title><path class="cls-1" d="M26.06,157.2A52.18,52.18,0,0,1,78.18,105h95.56V79H78.18A78.27,78.27,0,0,0,0,157.16V301.73a78.27,78.27,0,0,0,78.18,78.18H243.72a78.27,78.27,0,0,0,78.18-78.18V222H295.83v79.75a52.18,52.18,0,0,1-52.13,52.13H78.18a52.18,52.18,0,0,1-52.12-52.13"/><path class="cls-1" d="M196.07,12.32A13,13,0,0,0,208.32,26h0L342.83,33.6,164.26,212.22a13,13,0,0,0,18.43,18.43h0L361.27,52.07l7.63,134.49a13,13,0,1,0,26-1.47h0L386.63,39A32.56,32.56,0,0,0,355.9,8.27L209.81,0a13,13,0,0,0-13.74,12.21Z"/></svg>
\ No newline at end of file
+<svg viewBox="0 0 394.91 379.91">
+  <path 
+    fill="currentColor"
```

Comment:
failed attempt to make the icon get the currentColor

Reply:
😂